snapshot-labs/sx-monorepo
# fix: fix hidden choice filter dropdown in votes page

### Summary



Closes: https://github.com/snapshot-labs/workflow/issues/239

Fix the choice filter dropdown not appearing on votes page, becuase of overflow hidden

This PR will use the Float library, to show the dropdown in a portal

### How to test

1. Go to a proposal with basic choice
2. Go to votes page
3. The choice filter dropdown should be  visible on click
